test/build-test-coverage/lcov-report/base.css

```diff
@@ -0,0 +1,213 @@
+body, html {
```

Comment:
This directory should be added to the gitignore shouldn't it? We ignore it in cfgov-refresh https://github.com/cfpb/cfgov-refresh/blob/master/.gitignore#L104